### PR TITLE
feature/SYUS-10542

### PR DESCRIPTION
**Pull request by ShervinM-AVISPL**
Replaced usage of `AVISymphonyDev` and `AVISymphonyDev@avispl.com` with secrets

### CODE REVIEW

File: actions/yaml_dummy_update_file_commit_push/action.yml [hunk 1]
```diff
@@ -46,6 +46,12 @@ inputs:
       Prefix must be unchanged otherwise old lines cannot be found
     required: false
     default: 'true'
+  commit-user-name:
+    description: User to be used as committer
```
Comment (SebastienB-AVISPL) [inline actions/yaml_dummy_update_file_commit_push/action.yml:50]: "...as Git committer"
(prefer to add Git for clarity, same for other places)

File: actions/yaml_dummy_update_file_commit_push/action.yml [hunk 2]
```diff
@@ -85,8 +91,8 @@ runs:
           commit_message: 'AUTO Update ${{ inputs.yaml-file  }}'
           # TODO Git commit user and author - use secrets instead of hardcoded value
```
Comment (SebastienB-AVISPL) [inline actions/yaml_dummy_update_file_commit_push/action.yml:92]: Remove TODO

File: actions/kubernetes_update_deployment/action.yml [hunk 3]
```diff
@@ -129,8 +135,8 @@ runs:
         commit_message: '${{ inputs.commit-message }}'
         # TODO Git commit user and author - use secrets instead of hardcoded value
```
Comment (SebastienB-AVISPL) [inline actions/kubernetes_update_deployment/action.yml:136]: remove TODO

**Review by SebastienB-AVISPL — CHANGES_REQUESTED** [review]

File: .github/workflows/build_branch_npm.reusable.workflow.yml [hunk 4]
```diff
@@ -83,6 +83,21 @@ on:
           It must be a GitHub PAT (Personal Access Token) with read and write authorizations
           on the GitHub repositories to be read/updated
         required: false
+      commitUserName:
+        description: |
+          User name to be used as Git committer. This is NOT the user name of the person who
```
[the remark this answers is not in the document]
Comment (ShervinM-AVISPL) [reply .github/workflows/build_branch_npm.reusable.workflow.yml]: done

File: actions/yaml_dummy_update_file_commit_push/action.yml (hunk 1 above)
Comment (ShervinM-AVISPL) [reply actions/yaml_dummy_update_file_commit_push/action.yml]: done

File: actions/yaml_dummy_update_file_commit_push/action.yml (hunk 2 above)
Comment (ShervinM-AVISPL) [reply actions/yaml_dummy_update_file_commit_push/action.yml]: done

File: actions/kubernetes_update_deployment/action.yml (hunk 3 above)
Comment (ShervinM-AVISPL) [reply actions/kubernetes_update_deployment/action.yml]: done

**Pull request merged**